media/css/firefox/family/index.less

```diff
@@ -253,8 +253,8 @@ html[dir="rtl"] #header h1 {
 /*--------------------------------------------------------------------------*/
 // Base styles for product groups
 .product-list {
-    .clearfix();
-    .flexbox();
+    .clearfix;
+    .flexbox;
```

Comment:
Oops, forgot to set `clear:none` to override the `clear:both` that applies at other sizes.